Makefile

```diff
@@ -67,7 +67,7 @@ FUZZER_BUILD ?= 0
 # NOTE: when changing any value below, you'll need to adjust WORKSPACE file by hand:
 #         - uncomment source build section, comment binary section
 #         - adjust binary version path - version variable is not passed to WORKSPACE file!
-OV_SOURCE_BRANCH ?= 5c0f38f83f62fdabcdc980fa6dc3ed1ea16c8a05  # 29 May releases/2024/2
+OV_SOURCE_BRANCH ?= 4d86daa2638eeecee57d62db04c18d6c59e5eb07  # 19 June main
```

Comment:
can you also update tokenizers and contrib to the same package using master branch?
contrib e6eb43a32c98a04162a921a80d89f82b30910973
0dcc805651b7744cf8340c4654cc3ea7523b28b8